tasks/mutations/patient_mutations.ts

```diff
@@ -13,12 +13,12 @@ import {
   GetPatientAssignmentByWardRequest,
   GetPatientListRequest,
   DeletePatientRequest,
-  ReadmitPatientRequest
+  ReadmitPatientRequest, GetRecentPatientsRequest
```

Comment:
Please fix your formatter.

```suggestion
  ReadmitPatientRequest,
  GetRecentPatientsRequest
```